### DISCUSSION TITLE
copilot sign up issue

### DISCUSSION THREAD

**Discussion by nyteghost** [community/community#32435] [Copilot Conversations]
I signed up for the trial of copilot, and while I still had it, I was eligible to sign up for the student pack. I signed up through there, and now all it says is 
![image](https://user-images.githubusercontent.com/12069785/189396092-ccf035a1-b2e4-4f26-b1d6-4c1ea733db1f.png)
but it won't let me do anything passed that.

**Comment by iamcjwilliams** [comment]
@nyteghost Please visit [here](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student) for more information about applying to Global Campus as a student

**Reply by nyteghost** [reply]
I already have done that and was approved. I have access to the other items the student pack offers, but github copilot does not work correctly. I can not sign up for it, even sign up to pay for it.

**Reply by iamcjwilliams** [reply]
@nyteghost Since this is account-specific, a support ticket will need to be submitted. [Support page](https://support.github.com/contact).  Once a ticket is open, our support team will be able to get you signed up!
